pyanaconda/modules/payloads/payload/live_image/installation.py

```diff
@@ -144,18 +144,19 @@ def _stream_download(self, response, image_file):
         progress.end()
 
 
-class VerifyImageChecksum(Task):
+class VerifyImageChecksumTask(Task):
     """Task to verify the checksum of the downloaded image."""
 
-    def __init__(self, image_path, checksum):
+    def __init__(self, configuration: LiveImageConfigurationData, image_path):
```

Comment:
I wonder, do we want the type specified here? Isn't it enough to have it in the docstring?

Reply:
It is useful for development, if an IDE can do type checking. I can remove it, but I don't think it is so bothering.

Reply:
In Pycharm, the type checking happens also with data from docstrings - at least for me...

Reply:
That sometimes doesn't work and not all type checkers are able to use docstrings. For example, mypy that we would like to use in the future.

Reply:
I don't mind, I just wondered that it's just for this one argument and nothing else. Feel free to keep it here.